engine/cache.js

```diff
@@ -1,24 +1,82 @@
-import fs from 'fs';
-import path from 'path';
+// import fs from 'fs';
+// import path from 'path';
+// import fetch from 'node-fetch';
+// import fileExists from 'file-exists';
+// import crypto from 'crypto';
+
+import Eu from 'eu';
+import redis from 'redis';
 import fetch from 'node-fetch';
-import fileExists from 'file-exists';
-import crypto from 'crypto';
+import stream from 'stream';
 
-export default {
-  readJson: (src, cacheDir) => {
-    const shasum = crypto.createHash('sha1');
-    shasum.update(src);
-    const basename = shasum.digest('hex');
-    const cacheFilename = path.join(cacheDir, `${basename}.json`);
-    if (fileExists(cacheFilename)) {
-      return Promise.resolve(JSON.parse(fs.readFileSync(cacheFilename)));
+function readJson(url, fake, dbNumber) {
```

Comment:
it is to replace cacheDir - I wanted to give a grasp of how I think it can be implemented before changing everything